Lib/test/test_copy.py

```diff
@@ -945,6 +947,28 @@ def test_namedtuple(self):
         with self.assertRaisesRegex(ValueError, 'unexpected field name'):
             copy.replace(p, x=1, error=2)
 
+    def test_structseq(self):
```

Comment:
I think that it is better to test it in `test_structseq` module. Most `__replace__` implementations are tested together with other methods of their class. Only exceptions are `namedtuple` and `dataclass` because they are fabrics to create new classes.

It may be worth to split the test on two tests: with and without unnamed fields.